Assume 1000 hours for one failure, on average

In [1]:
1000.0 * 1 / 24

41.666666666666664

41.67 Days of computation per failure, on average

Now assume 200 computers all running, with the same mean time to failure.

In [2]:
(1.0 / 1000) * 200 * (24 / 1)

4.800000000000001

4.8 failures per day, once you have 200 computers running.